frontend/integration-tests-cypress/cypress-e2e/cypress/support/commands.js

```diff
@@ -0,0 +1,72 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add("login", (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add("drag", { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This is will overwrite an existing command --
+// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+import { appHost } from '../support/index';
+Cypress.Commands.add('getTestElement', (selector, timeout=4000) => cy.get(`[data-test-id="${selector}"]`, { timeout: timeout }));
+Cypress.Commands.add('getTestAction', (selector) =>
+  cy.get(`[data-test-action="${selector}"]:not(.pf-m-disabled)`),
+);
```

Comment:
I'd be inclined to split this file up into several files as it contains helper funcs for several contexts.  That should encourage us to stay organized and avoid a dump ground `utils` file.

Reply:
Not sure how to do that.  Cypress reads in `commands.js` can  I `import` other files which contain `Cypress.Commands.add(...)`, or maybe `import` init functions in other js files but execute them from `commands.js`?

Reply:
@benjaminapetersen No problem, we can import any named command file as long as we import it in `cypress/support/index.js`.